.github/workflows/codespell.yml

```
@@ -0,0 +1,17 @@
+name: Codespell Check
+
+on:
+  pull_request:
+    branches:
+      - gh-pages
+
+jobs:
+  codespell:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
```

We should use a newer version of the checkout command:

```suggestion
        uses: actions/checkout@v4
```